apps/mobile/src/libs/components/icon-button/icon-button.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import { MaterialIcon, Pressable } from "~/libs/components/components";
+
+type Properties = {
+	color: string;
+	name: string;
```

Comment:
The `string` doesn't suit here.
```suggestion
	name: ComponentPropsWithoutRef<typeof MaterialIcon>['name'];
```

Reply:
Where it could be better to place  `type ComponentPropsWithoutRef`, shared or mobile types? Don't know if frontend might use it.

Reply:
I don't like this type. We will have a separate IconName type for that